doc/conf.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
```

Comment:
I am fine with not including the encoding, but this was deleted without replacement again.